src/components/NewApiDelegationPage/NewApiDelegationPage.tsx

```diff
@@ -66,8 +95,20 @@ export const NewApiDelegationsPage = () => {
             <div className={classes.pageContent}>
               <h2>Gi tilgang til API</h2>
               <h3>Velg hvilke API du vil gi tilgang til ved å klikke på pluss-tegnet.</h3>
-              <div className={classes.searchField}>
-                <SearchField></SearchField>
+              <div className={classes.searchSection}>
+                <SearchField
+                  value={searchString}
+                  onChange={(e: any) => handleSearch(e.target.value)}
+                ></SearchField>
+                <div className={classes.filter}>
+                  <Select
+                    label={t('api_delegation.filter_label')}
+                    deleteButtonLabel={t('api_delegation.filter_remove_all')}
```

Comment:
I get the same problem as mentioned in comment above. Possible solution is same as above.